bitsdojo_window_linux/lib/src/bitsdojo_window_linux_real.dart

```diff
@@ -11,7 +11,7 @@ class BitsdojoWindowLinux extends BitsdojoWindowPlatform {
 
   @override
   void doWhenWindowReady(VoidCallback callback) {
-    WidgetsBinding.instance!.waitUntilFirstFrameRasterized.then((value) {
+    WidgetsBinding.instance.waitUntilFirstFrameRasterized.then((value) {
```

Comment:
You are breaking compatibility with older Flutter versions by doing this.
A better & expected change is given by the Flutter team itself in the release notes:
https://docs.flutter.dev/development/tools/sdk/release-notes/release-notes-3.0.0#your-code

Another solution can be to cast `WidgetsBinding.instance` into `WidgetsBinding`
```dart
(WidgetsBinding.instance as WidgetsBinding).waitUntilFirstFrameRasterized.then(...);
```

<hr></hr>

**Copied from link above.**

If the problem refers to your own code, you can update it by running dart fix --apply. This should resolve all the warnings.

If you need your code to support both Flutter 3 and earlier versions (maybe because your code is a library), then you can wrap calls to binding.instance with calls to a method such as the following:

```dart
/// This allows a value of type T or T?
/// to be treated as a value of type T?.
///
/// We use this so that APIs that have become
/// non-nullable can still be used with `!` and `?`
/// to support older versions of the API as well.
T? _ambiguate<T>(T? value) => value;
```
For example, instead of the following:
```dart
SchedulerBinding.instance!.addPostFrameCallback(...);
```
You can use:
```dart
_ambiguate(SchedulerBinding.instance)!.addPostFrameCallback(...);
```
When you no longer need to support versions of Flutter before 3.0.0, you can remove this and replace it with the following:
```dart
SchedulerBinding.instance.addPostFrameCallback(...);
```